### PR TITLE
fix(ui5-upload-collection): make empty upload collection focusable

### PR DESCRIPTION
**Pull request by dimovpetar**

### CODE REVIEW

File: packages/fiori/src/themes/UploadCollection.css [hunk 1]
```diff
@@ -11,6 +11,10 @@
 	background-color: var(--sapGroup_ContentBackground);
 }
 
+.uc-no-files-illustration {
+	pointer-events: none;
```
Comment (TeodorTaushanov) [inline packages/fiori/src/themes/UploadCollection.css:15]: It is strange that the text inside the illustrated message cannot be selected.

**Review by TeodorTaushanov — CHANGES_REQUESTED** [review]
The "cursor" should be the "default", instead of "pointer".

**Comment by dimovpetar** [comment]
> The "cursor" should be the "default", instead of "pointer".

Fixed

File: packages/fiori/src/themes/UploadCollection.css (hunk 1 above)
Comment (dimovpetar) [reply packages/fiori/src/themes/UploadCollection.css]: Removed `pointer-events: none`. As you proposed, this should be fixed in the list item by using delegatesFocus.

**Review by TeodorTaushanov — APPROVED** [review]

**Pull request merged**